vanetza/geonet/router.cpp

```diff
@@ -398,7 +398,6 @@ void Router::indicate_basic(IndicationContextBasic& ctx)
             indicate_secured(ctx, *basic);
         } else if (basic->next_header == NextHeaderBasic::Common) {
             if (!m_mib.itsGnSecurity || SecurityDecapHandling::Non_Strict == m_mib.itsGnSnDecapResultHandling) {
-                indication.security_report = security::DecapReport::Unsigned_Message,
```

Comment:
No, `security_report` is not optional in `geonet::DataIndication` though EN 302 636-4-1 says so. Why do you think that `Unsigned_Message` is unsuitable?

Reply:
Oh, sorry, that commit/change needs to be moved to #105. There, `security_report` is made optional because of EN 302 636-4-1 and EN 302 636-5-1.